Dockerfile

```diff
@@ -4,9 +4,7 @@ COPY requirements.txt /requirements.txt
 
 RUN pip install -r /requirements.txt
 
-RUN apt-get update \
- && apt-get install curl -y \
- && apt-get remove curl -y \
```

Comment:
Not sure what the purpose was of installing and then removing curl, but the golden tests pass without these lines.

Reply:
Thanks for the catch!  I think I cargo-culted these from a prior version that used Alpine.  😅